apps/aechannel/src/aesc_client_reconnect_tx.erl

```diff
@@ -0,0 +1,185 @@
+-module(aesc_client_reconnect_tx).
+
+-behavior(aetx).
+-behavior(aesc_signable_transaction).
+
+%% Behavior API
+-export([
+          new/1
+        , type/0
+        , fee/1
+        , gas/1
+        , ttl/1
+        , nonce/1
+        , origin/1
+        , check/3
+        , process/3
+        , signers/2
+        , version/1
+        , serialization_template/1
+        , serialize/1
+        , deserialize/2
+        , for_client/1
+        , valid_at_protocol/2
+        ]).
+
+-export([
+          channel_id/1
+        , channel_pubkey/1
+        , role/1
+        , round/1
+        , state_hash/1
+        ]).
+
+-define(CHANNEL_CLIENT_RECONNECT_TX_TYPE, channel_client_reconnect_tx).
+-define(CHANNEL_CLIENT_RECONNECT_TX_FEE, 0).  % off-chain
```

Comment:
Please use a double %% instead of a single %.

Reply:
The old convention for trailing comments is otherwise a single `%` ...
http://www.erlang.se/doc/programming_rules.shtml#HDR36